applications/nrf_desktop/configuration/nrf54l15dk_nrf54l10_cpuapp/hid_keyboard_leds_def_keyboard.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "hid_keyboard_leds.h"
+
+/* This configuration file is included only once from hid_state module and holds
+ * information about LEDs associated with HID keyboard LEDs report.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} hid_keyboard_leds_def_include_once;
+
+/* On the nRF54L10 SoC, you can only use the **GPIO1** port for PWM hardware peripheral output.
+ * Because of that, on the DK PCA10156 revision v0.8.1, **LED 0** and **LED 2** cannot be used for
+ * PWM output.
+ *
+ * You can still use these LEDs with the PWM LED driver, but you must set the LED color to
+ * ``LED_COLOR(255, 255, 255)`` or ``LED_COLOR(0, 0, 0)``. This ensures the PWM peripheral is not
```

Comment:
```suggestion
 * `LED_COLOR(255, 255, 255)` or `LED_COLOR(0, 0, 0)`. This ensures the PWM peripheral is not
```
Single backticks are enough in headers.